terraso_backend/config/settings.py

```diff
@@ -44,6 +44,7 @@
     "apps.graphql",
     "apps.auth",
     "apps.shared_data",
+    "geopandas",
```

Comment:
questions: Why is necessary to add `"geopandas"` here, is there some kind of integration with Django?

Reply:
@david-code I followed a tutorial online, that is why I added it here, I will check if I can remove it